src/control/controlbehavior.cpp

```diff
@@ -93,13 +103,23 @@ double ControlPotmeterBehavior::valueToMidiParameter(double dValue) {
     }
 }
 
+double ControlPotmeterBehavior::scaleStartParameter() {
+    return m_dScaleStartParameter;
+}
+
+void ControlPotmeterBehavior::setScaleStartParameter(double value) {
+    // TODO(Ferran Pujol): Check if value is between valid bounds (0,1)?
```

Comment:
What about this todo? Or is it a better place to check the bounds where bad things can happen?